src/components/accordion/_macro.spec.js

```diff
@@ -4,64 +4,82 @@ import * as cheerio from 'cheerio';
 
 import axe from '../../tests/helpers/axe';
 import { renderComponent } from '../../tests/helpers/rendering';
-
-const EXAMPLE_ACCORDION_WITH_TWO_ITEMS = {
-    id: 'accordion-identifier',
-    itemsList: [
-        {
-            title: 'Title for item 1',
-            content: 'Content for item 1',
-        },
-        {
-            title: 'Title for item 2',
-            content: 'Content for item 2',
-        },
-    ],
-};
-
-const EXAMPLE_ACCORDION = {
-    ...EXAMPLE_ACCORDION_WITH_TWO_ITEMS,
-    allButton: {
-        open: 'Open label',
-        close: 'Close label',
-    },
-};
-
-describe('macro: accordion', () => {
-    it('passes jest-axe checks', async () => {
-        const $ = cheerio.load(renderComponent('accordion', EXAMPLE_ACCORDION));
-
-        const results = await axe($.html());
-        expect(results).toHaveNoViolations();
+import { EXAMPLE_ACCORDION } from './_test_examples';
+
+describe('FOR: Accordion', () => {
```

Comment:
```suggestion
describe('FOR: Macro: Accordion', () => {
```
Think we should make this change to specify between macro and script tests. Making this change and the same for scripts will allow us to still specify if we want to run all script tests, all macro tests, all tests or the tests for a single component